main/debootstrap/APKBUILD

```diff
@@ -1,24 +1,24 @@
 # Contributor: Sören Tempel <soeren+alpine@soeren-tempel.net>
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=debootstrap
-pkgver=1.0.96
+pkgver=1.0.97
 pkgrel=0
 pkgdesc="Debian/Ubuntu bootstrap scripts"
 url="https://packages.qa.debian.org/d/debootstrap.html"
 arch="noarch"
-license="GPL-2.0"
+license="MIT"
 options="!check" # upstream doesn't have a test suite
 depends="debian-archive-keyring dpkg tar"
 makedepends=""
 subpackages="$pkgname-doc"
 source="http://ftp.debian.org/debian/pool/main/d/${pkgname}/${pkgname}_${pkgver}.tar.gz"
-builddir="$srcdir"/$pkgname-$pkgver
+builddir="$srcdir/$pkgname-$pkgver"
```

Comment:
Can you explain why you are making these changes?
If it doesn't make much sense please don't clobber patches with it.

Reply:
@clandmeter Sure.
I extended the double quote to include all variables in case of empty spaces.
Reverted

Reply:
While you are right regarding spaces both pkgname and pkgver can never include spaces so this should never happen.